scripts/prepare-preview-builds.sh

```diff
@@ -45,6 +45,18 @@ yarn install --no-immutable
 
 echo "Updating TypeScript imports for specific packages..."
 
```

Comment:
I actually forgot we needed this for publish previews to work it's definitely not scalable but essentially it goes through and replaces all the instances of inter dependent dependencies with `@metamask-previews/`

Reply:
Moved to this PR: https://github.com/MetaMask/metamask-design-system/pull/822